composer.json

```
@@ -39,12 +39,19 @@
         "irc": "irc://irc.freenode.org/muffin"
     },
     "require": {
-        "cakephp/orm": "^4.5"
+        "cakephp/orm": "^5.0"
     },
     "require-dev": {
-        "cakephp/cakephp": "^4.5",
-        "cakephp/cakephp-codesniffer": "^4.2",
-        "phpunit/phpunit": "^9.3"
+        "cakephp/cakephp": "^5.0",
+        "cakephp/cakephp-codesniffer": "^5.0",
+        "phpunit/phpunit": "^10.1",
```

```suggestion
        "phpunit/phpunit": "^10.5.5",
```
This is what the core has.

---

Makes sense, updated it.